packages/main/test/pages/TabContainer.html

```diff
@@ -431,6 +435,20 @@ <h2>Tab Container With Custom Menu Button</h2>
 		</ui5-tabcontainer>
 	</section>
 
+	<section>
+		<ui5-tabcontainer id="tabContainerNoContentPadding">
```

Comment:
The id tabContainerNoContentPadding doesn't seem appropriate because we override the padding in this example. How about tabContainerCustomContentPadding?

Reply:
I agree the ID of the tab container is misleading a bit, but it's in the sample. I don't mind that much, but ideally it should be: tabContainerCustomContentPadding

Reply:
done